showpass/Showpass-E2E-tests
# TA-56 verify popular city name on cards

**Description:**
This script will be verifying that when a popular city is selected, events showing up as cards contain the correct city.
Steps:
- Navigate to [Create Amazing Experiences](https://beta.showpass.com/) 
- Click on a popular city Calgary | Vancouver | Toronto
- Ensure the correct city name shows up on all the carousel event cards for that city

**Jira ticket:**
https://showpass.atlassian.net/browse/TA-56

**How to test:**
Run the following test suite and ensure it passes
cypress/e2e/home/popular-cities.cy.js

**Test execution results:**

<img width="984" alt="TA-56-all-tests-passed" src="https://github.com/showpass/Showpass-E2E-tests/assets/138615449/7fd0f450-c07b-4f31-ad6b-09e6550b27e8">
